### DISCUSSION TITLE
Maximum Execution time exceeded!!!

### DISCUSSION THREAD

**Discussion by Kuraj11101** [laravel/framework#33080] [General]
- Laravel Version: 7.11
- PHP Version: ^7.3.17
- Database Driver & Version: MySQL 

### Description:
Have had issues with user registration using Laravel passport and Jwt Token. I always get maximum execution time exceeded. At first it was 30secs I later change it to 300secs in my PHP.ini file but I still do have the same issue. And it have been over a week now!!!!

### Steps To Reproduce:
Create login and registration controller, create MySQL data with the necessary env configurations. Php artisan make:Auth and try to register a user.

**Comment by mrcat323** [comment]
But in laravel 7 `artisan make:auth` doesn't exist, how did you do it ?
Use [laravel/ui](https://github.com/laravel/ui) package, say `artisan ui bootstrap --auth` and try again. I tried, everything works fine

**Reply by Kuraj11101** [reply]
I am still getting the same error. 
![image](https://user-images.githubusercontent.com/58647906/83627451-78da6200-a586-11ea-9175-50758aac39de.jpg)
![image](https://user-images.githubusercontent.com/58647906/83627598-b2ab6880-a586-11ea-88f0-0ac4eb592887.jpg)

**Comment by driesvints** [comment]
Moved to discussions.

**Reply by Kuraj11101** [reply]
Ok the timeout error is gone. But user is not registering. I guess I have some debugging to with my controllers, routes and ProviderServices..

**Reply by Kuraj11101** [reply]
However I tried testing the api endpoint using Postman but still getting error. 
![image](https://user-images.githubusercontent.com/58647906/83642572-8b609580-a59e-11ea-9734-9487961cdc07.jpg)
![image](https://user-images.githubusercontent.com/58647906/83642604-961b2a80-a59e-11ea-83e2-c8e8dafb920f.jpg)
